src/ui/components/SuggestedResults/style.scss

```diff
@@ -0,0 +1,34 @@
+@import "~ui/css/vars";
+$item-padding: 5px 52px;
+
+.SuggestedResults {
+  background-color: $white;
+  color: $link-color;
+  padding-bottom: 10px;
+  padding-top: 10px;
+
+  &-list {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &-item {
+    color: inherit;
+    cursor: pointer;
+    outline: none;
+    padding: $item-padding;
+    text-align: left;
+    width: 100%;
+
+    &:hover,
+    &:active,
+    &:focus {
+      background-color: $background-gray;
+    }
+
+    .is-highlighted {
```

Comment:
I think just `highlighted` is fine, though we've tended to use very-scoped (and admittedly verbose) selectors for things like this: `.SearchResults-item--highlighted` is the usual approach, the double dashes signifying it's a utility/modifier instead of a different element.